src/util/piped_process.h

```diff
@@ -6,11 +6,10 @@
 #define CPROVER_UTIL_PIPED_PROCESS_H
 
 #ifdef _WIN32
-// TODO: Windows definitions go here
-#else
+#  include <windows.h>
```

Comment:
:no_entry_sign: Including `windows.h` in this header file exposes the entire windows API to any other file which includes this one when building on Windows. This has the potential to make it more difficult for contributors working on Windows to write platform independent code and could also increase build times. This issue could be worked around by forward declaring `PROCESS_INFORMATION` and `HANDLE` and making the member fields pointers to these types, or by using the "Pointer to implementation" / "pImpl" technique to only declare fields of these types in the `.cpp` file.

Reply:
Fixed.